app/directives/accessControlForm/porAccessControlFormController.js

```diff
@@ -58,10 +58,13 @@ function ($q, UserService, Notifications, Authentication, ResourceControlService
     })
     .then(function success(data) {
       if (data) {
+        ctrl.formData.AccessControlEnabled = true;
```

Comment:
Why did you these 2 lines? This will cause the access control form to be disabled by default when adding a new container.